docs/autogenerated-menu/Our Practices/Engineering/Procedures/Document Demonstrate Duplicate.md

```diff
@@ -0,0 +1,41 @@
+
+It’s about getting what’s in your head out into the world in a way someone else can follow. And not just kind of follow — actually do the thing well, without needing you standing over their shoulder.
+
+It’s not fancy. It just works. And once you’ve used it a few times, it becomes second nature — a fast, simple way to hand things over with confidence and clarity. That means fewer mistakes, faster onboarding, and much higher confidence in handovers and training.
+
+## Document. 
+You make a guide. You already know how to do the thing. Now you just need to write down the steps exactly as you do it. 
+_(Document and Demonstrate can be flipped around by having other trusted observers watch you and document what you do.)_ 
+Where possible record doing the thing multiple ways and in multiple scenarios. This way, you can watch yourself as an observer rather than breaking your flow by pausing to take notes while you go. Once you've got everything put into the guide, use it. Follow your own steps. Can you do an A+ job only following your directions exactly? If you can, you have the first draft of your guide for the job. 
+
+## Demonstrate: 
+You do it in front of the person you are training/handing over to. You sit down and walk them through the guide step by step. This may take a while depending on how many steps it takes to complete the thing. _If they stop you, or slow you down to understand something, adjust your guide for that._
+Now you have the second draft ready for them to try. 
+ 
+
+## Duplicate: 
+They do it in front of you. Now it's their turn. They follow the same guide you followed. Except this time, they're the one doing, and you're the one observing. We just want them to duplicate what we did. So if the guide is right, the outcome will be the same. And if the guide is off, you'll find out fast! Fix your guide until it's right. Then, have them follow it until they get it right. And once they nail it, high fives all round! _(High fives are non-negotiable.)_
+
+After you train or handover a few times this way, you’ll have worked out the kinks for that job and it’s pretty smooth sailing from there. 
+_Think about it this way, if you vanished tomorrow, could a new person get the results you get if they only followed your guide? That’s the level of clarity to shoot for._
+
+## Training Notes
+
+A helpful way to look at this training style is: If they get it wrong or get confused then we got it wrong or made it confusing. If we have to explain what a step means then the step is too complicated. Or, more likely, we tried to put multiple steps into one.
+
+If they only appear to “get it” after a longish explanation or multiple demonstrations then, again, we’ve got some work to do. Anyone that ignore this run into chronic training problems. And, word to the wise, you can probably force an inferior guide to work, but this turns into a nightmare when somebody else takes over your training for you.   
```

Comment:
```suggestion
If they only appear to “get it” after a longish explanation or multiple demonstrations then, again, we’ve got some work to do. Anyone that ignores this will run into chronic training problems. And, word to the wise, you can probably force an inferior guide to work, but this turns into a nightmare when somebody else takes over your training for you.   
```